src/components/votingcenter/CenterList.js

```diff
@@ -6,8 +6,10 @@ function CenterList() {
   const { state } = useContext(AppContext);
 
   const renderCenterList = () => {
-    return state.votingCenters.map((location) => (
+    return state.votingCenters.map((location, index) => (
       <CenterListItem
+        id={index}
+        name={location.address.locationName}
```

Comment:
for now, I'm mocking the ID with the index, definitely can remove this later
I also added the name as a prop so that it can be grabbed more easily